samples/Azure.AI.DocumentTranslation/Generated/DocumentTranslationRestClient.cs

```diff
@@ -0,0 +1,1283 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Azure;
+using Azure.Core;
+using Azure.Core.Pipeline;
+
+namespace Azure.AI.DocumentTranslation
+{
+    /// <summary> The DocumentTranslationRest service client. </summary>
+    internal partial class DocumentTranslationRestClient
+    {
+        private string endpoint;
+        private ClientDiagnostics _clientDiagnostics;
+        private HttpPipeline _pipeline;
+
+        /// <summary> Initializes a new instance of DocumentTranslationRestClient. </summary>
+        /// <param name="clientDiagnostics"> The handler for diagnostic messaging in the client. </param>
+        /// <param name="pipeline"> The HTTP pipeline for sending and receiving REST requests and responses. </param>
+        /// <param name="endpoint"> Supported Cognitive Services endpoints (protocol and hostname, for example: https://westus.api.cognitive.microsoft.com). </param>
+        /// <exception cref="ArgumentNullException"> <paramref name="endpoint"/> is null. </exception>
+        public DocumentTranslationRestClient(ClientDiagnostics clientDiagnostics, HttpPipeline pipeline, string endpoint)
+        {
+            this.endpoint = endpoint ?? throw new ArgumentNullException(nameof(endpoint));
+            _clientDiagnostics = clientDiagnostics;
+            _pipeline = pipeline;
```

Comment:
Do we need to store an `apiVersion`?

Reply:
I think the code we are generating here is correct.  The TA services don't use an `apiVersion` parameter, instead the version is part of the resource path, and encoded as part of the [`hostTemplate`](https://github.com/Azure/azure-rest-api-specs/blob/3196a62202976da192d6da86f44b02246ca2aa97/specification/cognitiveservices/data-plane/TranslatorText/preview/v1.0-preview.1/TranslatorBatch.json#L1545).

Reply:
We emit the [same shape of code](https://github.com/Azure/azure-sdk-for-net/blob/e602d34b34084d4fff731f237b7a84fd3acaeca2/sdk/translation/Azure.AI.Translation.Document/src/Generated/DocumentTranslationRestClient.cs#L32-L36) from the Track 2 Data Plane generator for this service in `azure-sdk-for-net`.